.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
-- 0.1
+- "0.10"
```

Comment:
I think so. Just was waiting to let any feedback roll in.

Since it seems to be not straight forward to keeping it working, I'll remove support for node 0.10.